content/fr/docs/reference/_index.md

```diff
@@ -35,7 +35,7 @@ Pour appeler l'API de Kubernetes depuis un langage de programmation on peut util
 
 ## Documents de Référence des outils en ligne de commande (CLI)
 
-* [kubectl](docs/reference/command-line-tools-reference/kubelet/) - Principal outil en ligne de commande (CLI) pour exécuter et gérer un cluster Kubernetes.
+* [kubectl](/docs/reference/command-line-tools-reference/kubelet/) - Principal outil en ligne de commande (CLI) pour exécuter et gérer un cluster Kubernetes.
```

Comment:
it should redirect to /docs/reference/kubectl/overview/ instead (see the english version https://kubernetes.io/docs/reference/)

Reply:
Hm, miss it.